charts/cluster-scanner/templates/configmap.yaml

```diff
@@ -13,9 +13,6 @@ data:
   eve_enabled: {{ .Values.eveEnabled | quote}}
   eve_integration_enabled: {{ .Values.eveIntegrationEnabled | quote }}
   logging_level: {{ .Values.global.loggingLevel }}
-{{- if eq .Values.global.scannerMode "local" }}
-  local_registry_secrets: {{ include "cluster-scanner.runtimeStatusIntegrator.localCluster.localSecrets" . }}
-{{- end }}
```

Comment:
I am not sure why we removed this. In the `deployment.yaml` this config map still exists. :thinking:

Reply:
I've moved inside `cluster-scanner.modeConfig` helper func, which is used in the next line!

I though it was a good refactor so to have less places where `.Values.scannerMode` is checked

Reply:
oh you're right, I missed that `cluster-scanner.modeConfig` was included in the next line. Got it, thanks!